# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpBackgroundChannelChild.h
# Commit: 73b51a139c91
# Full Hash: 73b51a139c915c7e4aa7b9bff58769bd21cfcd78
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-24 09:31:07
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P5 allow OnProgress/OnStatus going thru pBg IPC, r=mayhemer,necko-reviewers
#   
#   Depends on D76971
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D76972
# ==============================================================================

diff -r 2a9873ad6856 -r 73b51a139c91 netwerk/protocol/http/HttpBackgroundChannelChild.h
--- a/netwerk/protocol/http/HttpBackgroundChannelChild.h	Tue Jun 23 16:56:53 2020 +0000
+++ b/netwerk/protocol/http/HttpBackgroundChannelChild.h	Tue Jun 23 16:57:10 2020 +0000
@@ -69,6 +69,11 @@
       const nsHttpHeaderArray& aResponseTrailers,
       const nsTArray<ConsoleReportCollected>& aConsoleReports);
 
+  IPCResult RecvOnProgress(const int64_t& aProgress,
+                           const int64_t& aProgressMax);
+
+  IPCResult RecvOnStatus(const nsresult& aStatus);
+
   IPCResult RecvFlushedForDiversion();
 
   IPCResult RecvDivertMessages();